airbyte-webapp/src/pages/SettingsPage/pages/ConnectorsPage/components/CreateConnectorModal.tsx

```diff
@@ -116,18 +120,18 @@ const CreateConnectorModal: React.FC<IProps> = ({ onClose, onSubmit, errorMessag
             dockerRepository: "",
           }}
           validateOnBlur
-          validateOnChange
+          validateOnChange={false}
```

Comment:
What's the reason for disabling this? Seems like the default `true` should be fine, since the error message is only shown if the field is `touched` (which happens on blur). Actually both `validateOnBlur` and `validateOnChange` default to `true`, so they could both be removed unless they are explicitly being set to `false`.

Reply:
Yep neither of those lines are needed.  Removed.